R/build-home-index.R

```diff
@@ -12,7 +12,10 @@ build_home_index <- function(pkg = ".", quiet = TRUE) {
   data <- data_home(pkg)
 
   if (is.null(src_path)) {
-    data$index <- linkify(pkg$desc$get("Description")[[1]])
+    data$index <- linkify(pkg$desc$get_field(
+      key = "Description",
+      default = NA_character_
```

Comment:
I think a default value of `""` would be better, you don't need to spell out the argument names.